### ISSUE TITLE
Few XMBC plugins in the patchstick, how to load others?

### ISSUE THREAD

**Issue by GoogleCodeExporter** [davilla/atvusb-creator#216]
```
What steps will reproduce the problem?
1. open plugins from the video menu
2. only itunes video previews lists there

What is the expected output? What do you see instead?
- I'd expect a dozen or so different plugins

What version of the product are you using? On what operating system?
- aple tv, latest patchstick

Please provide any additional information below.
- so how to add plugins? where from? should the ones written for Windows
version of XBMC referring to drive letters like Q: would out of the box?

In short, would be nice to have a few more plugins ready in the
distribution, or at least a plugin loader.
```

Original issue reported on code.google.com by `ziga.turk` on 5 Jan 2009 at 7:35
